Cargo.toml

```diff
@@ -56,7 +56,7 @@ name = "snarkos"
 path = "snarkos/main.rs"
 
 [features]
-metrics = [ "snarkos-node-metrics", "snarkos-node/metrics" ]
+default = [ "snarkos-node-metrics", "snarkos-node/metrics", "snarkos-cli/metrics" ]
```

Comment:
Adding `snarks-cli/metrics` here isn't strictly necessary for the feature to be compiled in (as they are additive across shared dependencies) but it is more complete.